.github/workflows/pr-validation.yml

```diff
@@ -114,8 +114,8 @@ jobs:
       - name: Build docker image & build configuration YAML
         run: |
           container_id=${{steps.devcontainer.outputs.id}}
-          docker exec "$container_id" task controller:docker-build-and-save
-          docker exec "$container_id" task controller:run-kustomize-for-envtest
+          docker exec "$container_id" task controller:docker-build-and-save --concurrency 1 # limit concurrency because build machines are small
```

Comment:
Wouldn't this hit timeout/take too much time while running the tests in parallel?

Reply:
If our build machines were highly spec'd, limiting concurrency to just one task at a time would leave processors idle and memory unused. But we know from experience that the build machines are small, just two vCPU's, and so my hypothesis is that running too many tasks at the same time results in them fighting for resources and everything gets slowed down.